### ISSUE TITLE
Graphs constructed from EEG data

### ISSUE THREAD

**Issue by sgomezr** [openconnectome/Statistical-Connectomics-Class#230]
How do you compute a network from EEG recordings? fMRI is far more popular for constructing networks, then why use EEG? Can you really trust your networks constructed using EEG data? I'm just skeptical about them....

**Comment by kristinmg** [comment]
There are probably some other ways too, but I have been working with EEG data where we used each electrode as a node in the network and used some predefined measures (for example cosine similarity) to calculate/determine the connectivity between two nodes in the network.

**Comment by sgomezr** [comment]
But how do you process the data? Like, for fMRI, MRI you correct for Eddy currents, subject movement, etc.. what do you correct/normalize for when you're using EEG? And how do you guys relate each node (defined as an electrode) with the corresponding brain region?

**Comment by indivaux** [comment]
If you're using external EEG, my understanding is that because of the noise associated with passing through the skull, the spatial resolution isn't much more useful than identifying the lobe that a signal is coming from. Unless someone's come up with a really clever algorithm recently. Is that what you're doing @kristinmg, or is what you're doing not dependent on spatial data?

**Comment by kristinmg** [comment]
For my data we're only using very few electrodes so it actually turned out not to be very useful to use networks based analysis. But you're right, there is a lot of noise at least for scalp EEG. For what I was doing we only referred to the location of the electrodes, named after the corresponding lobe (frontal, occipital etc..).

**Comment by yaxigeigei** [comment]
I recently read a paper that reviews two frameworks for analyzing macroscopic connectivity of functional brain networks. These methods are applicable to both fMRI and EEG/MEG data and the review contains examples on all of them. 

Analysing connectivity with Granger causality and dynamic causal modelling
Current Opinion in Neurobiology, 2013
Karl Friston, Rosalyn Moran and Anil K Seth

**Comment by aceeccc** [comment]
I'm using intracranial EEG (SEEG) for some research work and the thing with fMRI is that it gives poor temporal resolution. For my research, I'm evaluating brain states through different tasks and the tradeoff of high spatial resolution from fMRI to high temporal resolution was worth it. (Also I didn't have much choice because the patients in this study had intracranial electrodes placed in order to treat their epilepsy...borrowing data...). SEEG also offered some advantages over regular EEG because the electrodes went inside the brain and were able to capture data from areas such as the hippocampus, instead of just remaining on the surface and capturing noisy signals.

In terms of computation, the EEG data I'm playing with gives power in specific frequency bands that you can build a network with by using some measure of similarity (eg. cosine similarity) as connection weight. With 8-12 nodes maybe the data wasn't the most significant, and I'm still analyzing the data, but when viewing our time series data and plotting 2 standard deviations, there were some significant differences in network eigenvector centrality of nodes....

**Comment by sgomezr** [comment]
Yeah, @indivaux mentioned SEEG last time we talked, I honestly didn't know they were used in humans, and they sound awesome (also, borrowing data can be good :) see it as a way of making the whole research process more efficient - you're maximizing  the conclusions/knowledge  obtained using a given data set).  I can see why you consider using SEEG  superior to using EEG,  you're measuring the activity of a specific region, and therefore can associate the signal of an electrode with a physiological node. So that case is different; the main reason I'm skeptical about studies with scalp EEG is that it seems to me that they have more 'noise' or error.... so I guess I should restate the question: when using scalp EEG, how do you usually process the data?

**Comment by indivaux** [comment]
I wonder what modifications can be made ta a graph to take into account this higher temporal resolution? Maybe the nodes can be less representative of the spatial arrangement and more represent the temporal pattern?

**Comment by SandyaS72** [comment]
For scalp EEG, I've done a little work with it - and yes you know going in that the spatial resolution isn't great...sometimes depending on what you're looking at, you might filter for a particular frequency band. Also the high temporal resolution becomes more helpful if you're thinking about a dynamic network... like during a task or a seizure or something... then you can build a network for different overlapping chunks of time and measure changes in structure or whatever metric you want through time. Connection weights usually come from some form of similarity or correlation between pairs of nodes.

**Comment by mblohr** [comment]
Do depth electrodes in conjunction with strip electrodes in ECoG, or SEEG, help with spatial resolution?

**Comment by akim1** [comment]
Pretty sure depth electrodes require the skull to be opened, so probably not used with scalp eeg (which is over the skin).
